src/Nethermind/Nethermind.Synchronization.Test/ParallelSync/MultiSyncModeSelectorTests.cs

```diff
@@ -506,7 +552,7 @@ public ScenarioBuilder PeersFromDesirableBranchAreKnown()
                 
                 public ScenarioBuilder PeersWithWrongDifficultyAreKnown()
                 {
-                    AddPeeringSetup("wrong difficulty", AddPeer(ChainHeadWrongDifficulty));
+                    AddPeeringSetup("wrong difficulty", AddPeer(ChainHeadWrongDifficulty), AddPeer(ChainHeadParentWrongDifficulty));
```

Comment:
should we split it into two different scenario s(or three?)

Reply:
I don't think it matters, both are ignorred